### DISCUSSION TITLE
A question about download speeds.

### DISCUSSION THREAD

**Discussion by shuttittuppitt** [TeamNewPipe/NewPipe#10456] [Questions]
Just now, I was trying to download a youtube video. I didn't start playing the video (in NewPipe), but instead started to download it first.. My download speed was between 150 to 200 kbs (sometimes much lower). After about 33%, I tried PLAYING the video (in NewPipe) to check the speed of my connection. When I did that, my download speed became an average of 2.10 to 2.80 mbs (sometimes a lot higher). This isn't the first time that I've noticed this either, but it never occurred to me to ASK about it before. Anyway, my question is do you know why that helped, because I assumed the opposite result would happen.

**Comment by opusforlife2** [comment]
Don't know about Newpipe, but generally, servers decide what speed to set for transfers in the initial few seconds, and sometimes they set it to something stupidly low like 10% of your available bandwidth, maybe because of random lags or throttling somewhere in between the two endpoints. I suspect that starting a second connection to the server causes it to reevaluate its previously set speed.
